setup.py

```diff
@@ -30,6 +30,13 @@
         'pytest-runner',
     ],
     classifiers=[
-        # TODO(Add items)
+        (
+            'License :: OSI Approved :: ',
+            'GNU General Public License v2 or later (GPLv2+)'
+        ),
```

Comment:
Also, with a comma, this is a tuple with two strs. I guess that was not intentional.